README.md

```diff
@@ -5,6 +5,22 @@ The LightStep distributed tracing library for C++.
 
 ## Installation
 
+The library supports being built in several configurations to support a variety of uses.  The default build configuration includes gRPC support for sending data to LightStep, or alternately the library may be configured without gRPC to use a user-defined transport library. The library can be built with support dynamic loading, for applications that support more than one OpenTracing-compatible tracer.
```

Comment:
> can be built with support dynamic loading

Think there's a typo here

> The library can be built with support dynamic loading

If users are going to use the library with dynamic loading, I'd first recommend they download the binary plugin attached to each release (https://github.com/lightstep/lightstep-tracer-cpp/releases/tag/v0.8.1).

It statically links in all the dependencies and uses an export map to hide most symbols so that they don't clash with those used by the app. The default shared library build will link to all the dependencies dynamically so it won't be portable to environments without those same dependencies installed.

Reply:
Got it.